_episodes/01-intro.md

```diff
@@ -79,17 +62,102 @@ the Bourne Again SHell
 Bash is the default shell on most modern implementations of Unix
 and in most packages that provide Unix-like tools for Windows.
 
-### Why bother?
-Using Bash or any other shell
-sometimes feels more like programming than like using a mouse.
-Commands are terse (often only a couple of characters long),
-their names are frequently cryptic,
-and their output is lines of text rather than something visual like a graph.
-On the other hand,
-with only a few keystrokes, the shell allows us to combine existing tools into 
-powerful pipelines and handle large volumes of data automatically. This automation
-not only makes us more productive but also improves the reproducibility of our workflows by 
-allowing us to repeat them with a few simple commands.
+### What does it look like?
+
+A typical shell command and output looks something like this:
+
+~~~
+bash-3.2$ 
+bash-3.2$ ls -F / 
+Applications/         System/
+Library/              Users/
+Network/              Volumes/
+bash-3.2$ 
+~~~
+{: .language-bash}
+
+The first line shows only a **prompt**, indicating that the shell is waiting
+for input. Your shell may use different text for the prompt. Most importantly: 
+when typing commands, either from these lessons or from other sources,
+*do not type the prompt*, only the commands that follow it.
+
+The part that you type, in this example `ls -F /` is made up of a **command**,
+some **flags** (also called **options**) and an **argument**. The main difference 
+between flags and arguments is that opitons start with a dash (`-`). Sometimes
```

Comment:
Options misspelled - but more to the point, it should probably be 'flags' instead.